packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -102,6 +102,11 @@ const createProjectTasks = ({ newAppDir }) => {
           fs.ensureDirSync(path.dirname(newAppDir))
         }
         fs.copySync(templateDir, newAppDir)
+        // .gitignore is renamed here to force file inclusion during publishing
+        fs.rename(
+          path.join(newAppDir, 'gitignore'),
```

Comment:
What do you think of naming it `gitignore.template`in the source so its obvious that it wasn't a typo in the filename?